<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kiryakov/Tymchenko</string>
  </property>
  <property name="toolTipDuration">
   <number>1</number>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread: pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(81, 0 135, 255), stop:0.005 rgba(51, 61, 132, 255), stop:1 rgba(155, 79, 165, 255));
font: 16pt &quot;Segoe UI Symbol&quot;;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>821</width>
      <height>421</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255);
border: 10px solid rgba(255, 255, 255, 60);
border-radius: 10px;</string>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="252">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>801</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
</string>
         </property>
         <property name="text">
          <string>                  SKO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
</string>
         </property>
         <property name="text">
          <string>               Variance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
</string>
         </property>
         <property name="text">
          <string>            Mat. waiting</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLCDNumber" name="cko">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 300);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
font: 20pt &quot;Arial&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="disp">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 300);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="mat">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 300);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>811</width>
      <height>82</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;</string>
         </property>
         <property name="text">
          <string>Type of values</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;</string>
         </property>
         <item>
          <property name="text">
           <string>Re</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Im</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Abs</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Phase</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Select a file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start">
         <property name="text">
          <string>START</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Адрес выбранного файла&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 120);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;</string>
         </property>
         <property name="text">
          <string>                    Address of the selected file:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 200);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;
font: 24pt &quot;Arial&quot;;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
